system-addon/test/unit/lib/LocalizationFeed.test.js

```diff
@@ -87,16 +87,31 @@ describe("Localization Feed", () => {
         too: TEST_STRINGS[DEFAULT_LOCALE].too
       });
     });
-    it("should use all default strings for unknown locale", () => {
-      const locale = "xyz";
+    it("should use multiple fallback strings before default", () => {
+      const locale = "ru";
       sandbox.stub(global.Services.locale, "negotiateLanguages")
-        .returns([locale]);
+        .returns([locale, "it", DEFAULT_LOCALE]);
+
       feed.updateLocale();
 
       assert.calledOnce(feed.store.dispatch);
       const arg = feed.store.dispatch.firstCall.args[0];
       assert.propertyVal(arg, "type", at.LOCALE_UPDATED);
       assert.propertyVal(arg.data, "locale", locale);
+      assert.deepEqual(arg.data.strings, {
+        foo: TEST_STRINGS[locale].foo,
+        too: TEST_STRINGS.it.too
+      });
+    });
+    it("should use all default strings for unknown locale", () => {
+      sandbox.stub(global.Services.locale, "negotiateLanguages")
+        .returns([DEFAULT_LOCALE]);
+      feed.updateLocale();
```

Comment:
I'd suggest whitespace above this line, to keep with "setup/execute/verify" sections.